### PR TITLE
#91: Fixed issue with encoding passwords correctly

### PR DESCRIPTION
**Pull request by tomuben**

### CODE REVIEW

**Review by morazow — CHANGES_REQUESTED** [review]

File: doc/changes/changes_7.1.0.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ t.b.d.
 ## Bug Fixes
 - #106: Fixed installation issue with OpenSSL already installed message
 - #93: Fixed cloned connection
+- #91: Encode password correctly
```
[the remark this answers is not in the document]
Comment (tomuben) [reply doc/changes/changes_7.1.0.md:16]: Done

File: R/EXADBI-connection.R [hunk 2]
```diff
@@ -204,6 +204,11 @@ dbCurrentSchema <- function(con, setSchema=NULL) {
   substr(odbc_key_value, key_size, nchar(odbc_key_value) - 1)
 }
 
+.encode_password <- function(pwd) {
+  #pwd <- gsub("}", "}}", pwd, perl = TRUE) #Not yet supported by Exasol ODBC driver.
```
[the remark this answers is not in the document]
Comment (tomuben) [reply R/EXADBI-connection.R:208]: I created #114 for this.

**Review by morazow — APPROVED** [review]
Looks good

**Review by morazow — APPROVED** [review]
:+1:

**Pull request merged**